### PR TITLE
Integrate Grunt with Travis

### PR DESCRIPTION
**Pull request by contactashish13**
#2228

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -38,8 +42,15 @@ branches:
...
   - bash bin/install-wp-tests.sh wordpress_test root '' localhost $WP_VERSION
 
-script: if [[ $PHPUNIT = "local" ]]; then ./vendor/bin/phpunit; else phpunit; fi
+script: 
+  - if [[ $GRUNT = "yes" && "$TRAVIS_PULL_REQUEST" != "false" ]]; then grunt; fi
+  - if [[ $PHPUNIT = "local" ]]; then ./vendor/bin/phpunit; else phpunit; fi
+
+after_failure:
+- cat "logs/phpcs.log"
+- cat "logs/jslogs.log"
```
Comment (EkoJR) [inline .travis.yml]: I believe this would need a new line at the end of file.

File: bin/init-grunt.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+COMPOSER_LOCATION=$(composer config home --global)
+export PATH="$COMPOSER_LOCATION/vendor/bin:$PATH"
+composer self-update
+
+composer global require "squizlabs/php_codesniffer"
+
+git clone -b master --depth 1 https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards.git "$HOME/wordpress-coding-standards"
+phpenv rehash
+phpcs --config-set installed_paths "$HOME/wordpress-coding-standards"
+phpenv rehash
```
Comment (EkoJR) [inline bin/init-grunt.sh]: As well as this would need a new line at the end of file.

File: PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -19,6 +19,12 @@ _Put an `x` in the boxes that apply. You can also fill these out after creating
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] I have added necessary documentation (if appropriate)
 
+## Testing instructions
+-Don't assume the tester knows the entire backstory of the issue, and don't force him/her to decipher the code to try and figure out what -it's doing or how to test it.
+-Do provide step by step instructions on how to test. 
+-Do note things to watch out for.
+-Do not what aspects the tester should try and break.
```
Comment (EkoJR) [inline PULL_REQUEST_TEMPLATE.md]: Any chance this has a typo?

File: bin/init-grunt.sh (hunk 2 above)
Comment (contactashish13) [reply bin/init-grunt.sh]: done

File: .travis.yml (hunk 1 above)
Comment (contactashish13) [reply .travis.yml]: done

File: PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (contactashish13) [reply PULL_REQUEST_TEMPLATE.md]: no idea what this is

Comment (EkoJR) [reply PULL_REQUEST_TEMPLATE.md]: Was it supposed to be "Do not do the aspects the tester should try and break."?

Comment (EkoJR) [reply PULL_REQUEST_TEMPLATE.md]: Or was it "Do note* what aspects..."?

Comment (EkoJR) [reply PULL_REQUEST_TEMPLATE.md]: Oh, i see now, that is a commit from Michael.

Disregard

**Pull request merged**